packages/frontend/netlify.toml

```diff
@@ -23,13 +23,13 @@
     TOKEN_CONTRACTS = 'berryclub.ek.near,wrap.near,6b175474e89094c44da98b954eedeac495271d0f.factory.bridge.near'
     RECAPTCHA_ENTERPRISE_SITE_KEY = "6LcpJ3EcAAAAAFgA-nixKFNGWMo9IG9FQhH4XjSY"
     RECAPTCHA_CHALLENGE_API_KEY = "6LeRzswaAAAAAGeS7mSasZ1wDcGnMcH3D7W1gy1b"
-    USE_CLOUDFLARE = "true"
+    USE_CLOUDFLARE = "false"
```

Comment:
Don't we want to leave this enabled for the existing configurations and only apply it to the new Netlify environments?

Reply:
The issue is that `netlify.toml` [will override any envs we set on the UI](https://docs.netlify.com/configure-builds/environment-variables/#app) so I enabled it on the Netlify UI for the current environments and added an explicit disable on our `netlify.toml`

Reply:
If `netlify.toml` overrides the site configuration, then wouldn't this be disabling it for `*.near.org` where we want it to remain on?

Is it possible to specify a new configuration in `netlify.toml` for the new sites or are we limited to `[build.environment]` and `[context.production.environment]`?

Reply:
> If netlify.toml overrides the site configuration, then wouldn't this be disabling it for *.near.org where we want it to remain on?

Good point, it means I cant set the explicit `false`. I removed it from the toml

> Is it possible to specify a new configuration in netlify.toml for the new sites or are we limited to [build.environment] and [context.production.environment]

I was looking for this on [the docs](https://docs.netlify.com/configure-builds/file-based-configuration/) but I couldn't find anything

Reply:
I just realized that now `USE_CLOUDFLARE=true` means that we aren't using Cloudflare 😂 

Good call setting it explicitly in Netlify, it'll be nice when we can remove all references to it 🎉